l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -793,7 +847,7 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
         uint16 _l2TxNumberInBatch,
         bytes32[] calldata _merkleProof
     ) external override onlyLegacyBridge {
-        bytes32 assetInfo = nativeTokenVault.getAssetInfoFromLegacy(_l1Asset);
+        bytes32 assetInfo = IL1NativeTokenVault(nativeTokenVault).getAssetInfoFromLegacy(_l1Asset);
```

Comment:
no need to convert types,

Reply:
updated

Reply:
Its still converting types
<img width="1011" alt="image" src="https://github.com/matter-labs/era-contracts/assets/34402761/a3f24d23-451d-4e33-a70c-7bda848cacf9">

Reply:
updated